README.md

```diff
@@ -137,3 +137,15 @@ claimed_amounts: Vec<u128>
 - Set max number of RewardPools when setting up StakePool. An optimization for organizations that know they will only ever want to distribute a single SPL Token as rewards to stakers.
 - Allow for locking rewards based on duration. This is another incentive mechanism for getting wallets to stake for longer durations.
 - Early withdraw penalties. Some % of locked tokens get sent to DAO for withdrawing.
+
+## Development
+
+### Release
+
+```
+export GIT_SHA=$(git rev-parse HEAD)
```

Comment:
What is this information for?

Reply:
oops, export was useless since it's done in line. The variables will put the commit hash and current git tag of the build when compiled properly.